packages/website/content/integrations/dnd-kit.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title: 'Dnd kit'
+slug: '/integrations/dnd-kit/'
+section: 'integrations'
+---
+
+We recommend using the excellent [dnd kit](https://dndkit.com/) library for adding drag and drop functionality to Forma 36 components.
+
```

Comment:
```suggestion
## Import
```bash
import { DndContext } from '@dnd-kit/core';
import {
  arrayMove,
  horizontalListSortingStrategy,
  SortableContext,
  useSortable,
} from '@dnd-kit/sortable';
import { CSS } from '@dnd-kit/utilities';
```
```

Reply:
Since imports are not visible in the examples below, would be nice to see what I need to import